boot/bootstate20.go

```diff
@@ -385,7 +385,11 @@ func (ks20 *bootState20Kernel) loadBootenv() error {
 
 	// find the bootloader and ensure it's an extracted run kernel image
 	// bootloader
-	bl, err := bootloader.Find("", nil)
+	opts := &bootloader.Options{
+		// we want extracted run kernel images for uc20
+		ExtractedRunKernelImage: true,
```

Comment:
we need a todo about renaming this flag, it seems to mean something slightly different that what we intended.  uboot doesn't have extracted run kernel images (it has multiple files extracted)

Reply:
yes agreed the name is a bit weird it's usage now is more "uc20 bootloader" than "extracted run kernel image bootloader"